include/swift/Frontend/ModuleInterfaceLoader.h

```diff
@@ -170,14 +172,16 @@ class ModuleInterfaceLoader : public SerializedModuleLoaderBase {
          ArrayRef<std::string> PreferInterfaceForModules = {},
          bool RemarkOnRebuildFromInterface = false,
          bool IgnoreSwiftSourceInfoFile = false,
-         bool DisableInterfaceFileLock = false) {
+         bool DisableInterfaceFileLock = false,
+         bool DisableImplicitSwiftModule = false) {
```

Comment:
There seem to be enough flags here to probably warrant a `ModuleInterfaceOptions` type. That can be done in a follow-up.

Reply:
Yeah, I agree.